src/test/java/Homework16.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+  @test
+    public void registrationNavigation() {
+ //     Added ChromeOptions argument below to fix websocket error
+      ChromeOptions options = new ChromeOptions();
+      options.addArguments("--remote-allow-origins=*");
+
+      WebDriver driver = new ChromeDriver(options);
+      driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+      String url = "https://qa.koel.app/";
+      driver.get(url);
+
+      WebElement registrationLink = driver.findElement(By.cssSelector("[href='registration']"));
+      registrationLink.click();
+
+      String registrationUrl = "https://qa.koel.app/registration";
+      Assert.assertEquals(driver.getCurrentUrl(), registrationUrl);
+
+      driver.quit();
```

Comment:
Hi,
I have checked your homework 16, but I not see you resolve this error as I mention in the previous comment. (Issue about @Test testng annotations & add className in the suites TestNG.xml)
Please update code, push to your github and then submit link pull request for us review, run check again to complete this task.

About your question `THe only problem I have now is that when I run it, koel will open and close quickly. Isn't koel supposed to stay open when you run test?`. At first step in line 14 to 21 to initialization chrome driver, open browser and get koel url. After that will come the steps to execute the action on the web. And at end step `driver.quit(); `browser will quit.

So browser always to open in the you run test.

By the way, selenium also support running test in headless mode (Headless browser is a program that emulates a browser but has no user interface)
You can use `options.addArguments("--remote-allow-origins=*");` or `options.setHeadless(true);` to run in the headless mode